frappe/templates/includes/login/login.js

```diff
@@ -291,17 +291,10 @@ var verify_token = function (event) {
 }
 
 var request_otp = function (r) {
-	$('.login-content').empty().append($('<div>').attr({ 'id': 'twofactor_div' }).html(
-		'<form class="form-verify">\
-			<div class="page-card-head">\
-				<span class="indicator blue" data-text="Verification">{{ _("Verification") }}</span>\
-			</div>\
-			<div id="otp_div"></div>\
-			<input type="text" id="login_token" autocomplete="off" class="form-control" placeholder={{ _("Verification Code") }} required="" autofocus="">\
-			<button class="btn btn-sm btn-primary btn-block" id="verify_token">{{ _("Verify") }}</button>\
-		</form>'));
-	// add event handler for submit button
-	verify_token();
+	$('.login-content').empty();
+	$('.login-content:visible').append($('<div>').attr({ 'id': 'twofactor_div' }).html('{{ _("Verification") }}<input type="text" id="login_token" autocomplete="off" class="form-control" placeholder={{ _("Verification Code") }} required="" autofocus="">{{ _("Verify") }}'));
```

Comment:
This seems to strip down the entire form only to remain one input tag sandwiched between two translated strings. Specific reasons for doing this? Also, add screenshots of before and after too!

Reply:
The screen shots would look the same.  The problem, as described in the issue, is that the multiple login-content classed tags exist (some are hidden) which causes multiple input tags with the ID "login_token" to also exist.  This breaks JavaScript code that attempts to retrieve the value of the input tag.  

In regards to semgrep errors, I'm happy to revert the last commit.  Do you have rights to ignore the error and allow the pull?

Reply:
I've reverted the commit and updated your branch. semgrep's false positives were fixed in https://github.com/frappe/frappe/pull/13329.

---

About the solution you've proposed, I have yet to test it. WIll get back on this.

Reply:
@conncampbell Please add screenshots irrespective. It helps with credibility and also helps if someone looks back at this PR years from now.

Reply:
Ok, I've added screen captures of the issue and the fix.